dev/airflow-github

```diff
@@ -173,6 +173,9 @@ def is_core_commit(files: List[str]) -> bool:
         for ignore in non_core_files:
             if file.startswith(ignore):
                 break
+            # Handle renaming. Form: {old_name} -> {new_name}
+            elif file.startswith("{") and file[1:].startswith(ignore):
```

Comment:
We might need to be a little more flexible here. I was thinking we should just pop the complete "old" and "new" path into the list of files, then do our comparison against that.

Reply:
good idea